From ff1e4c6c25e4d804f7b4635dbf8cd4d696beef0a Mon Sep 17 00:00:00 2001
From: Ronan Abhamon <ronan.abhamon@vates.fr>
Date: Wed, 24 Aug 2022 17:09:11 +0200
Subject: [PATCH 072/177] fix(LinstorJournaler): reset namespace when `get` is
 called

Otherwise, we can be in the wrong namespace and the key to find will be inaccessible.
---
 drivers/linstorjournaler.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/linstorjournaler.py b/drivers/linstorjournaler.py
index 3993f601..1e85ec96 100755
--- a/drivers/linstorjournaler.py
+++ b/drivers/linstorjournaler.py
@@ -107,6 +107,7 @@ class LinstorJournaler:
             )
 
     def get(self, type, identifier):
+        self._reset_namespace()
         return self._journal.get(self._get_key(type, identifier))
 
     def get_all(self, type):
